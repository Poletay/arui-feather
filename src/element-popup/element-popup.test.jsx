import React from 'react';
import { mount } from 'enzyme';

import ElementPopup from './element-popup';

describe('element-popup', () => {
    it('should render without problems', () => {
        let elPopup = mount(<ElementPopup uniqID='1'><span>TEST</span></ElementPopup>);

        expect(elPopup).toMatchSnapshot();
    });
});
