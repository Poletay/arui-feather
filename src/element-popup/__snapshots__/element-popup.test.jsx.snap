// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`element-popup should render without problems 1`] = `
<ElementPopup
  direction="bottom"
  forceDirection={false}
  trigger="click"
  uniqID="1"
>
  <div
    className="element-popup element-popup_theme_alfa-on-white"
    id="1"
  >
    <Link
      checked={false}
      className="element-popup__children"
      disabled={false}
      iconPosition="left"
      onClick={[Function]}
      pseudo={true}
      size="m"
      tabIndex={0}
      url="#"
    >
      <a
        className="link link_pseudo link_size_m link_theme_alfa-on-white element-popup__children"
        href="#"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        tabIndex={0}
      >
        <span>
          TEST
        </span>
      </a>
    </Link>
    <Mq
      query="--medium"
    />
    <Mq
      query="--small-only"
    />
  </div>
</ElementPopup>
`;
